```
21:28:10 ownCloud is not installed - only a limited number of commands are available
21:28:10 Oracle connection could not be established
21:28:10  -> ORA-01033: ORACLE initialization or shutdown in progress Check environment: ORACLE_HOME=/usr/lib/oracle/12.1/client64 ORACLE_SID= LD_LIBRARY_PATH= NLS_LANG= tnsnames.ora is not readable
21:28:10 Kill the docker 82a71b6bb7a14679c2ae62dfd3d1a96217c990f728da441f48fe2e2e92b31a05
21:28:19 82a71b6bb7a14679c2ae62dfd3d1a96217c990f728da441f48fe2e2e92b31a05
21:28:19 82a71b6bb7a14679c2ae62dfd3d1a96217c990f728da441f48fe2e2e92b31a05
21:28:19 Makefile:179: recipe for target 'test-php' failed
21:28:19 make: *** [test-php] Error 1
```

--- a/apps/files_trashbin/lib/Trashbin.php
+++ b/apps/files_trashbin/lib/Trashbin.php
@@ -183,6 +183,11 @@ class Trashbin {
 		$target = $user . '/files_trashbin/files/' . $targetFilename . '.d' . $timestamp;
 		$source = $owner . '/files_trashbin/files/' . $sourceFilename . '.d' . $timestamp;
 		self::copy_recursive($source, $target, $view);
+
+		if ($view->file_exists($target)) {
+			self::insertTrashEntry($user, $targetFilename, $targetLocation, $timestamp);
+			self::scheduleExpire($user);
+		}
 	}
 
 	/**

